wazo_calld/plugins/dial_mobile/bus_consume.py

```diff
@@ -50,6 +60,7 @@ def _on_user_event(self, event):
             video_enabled,
             ring_timeout,
             event["Linkedid"],
+            creation_time=timestamp,
```

Comment:
I find the key name `creation_time` not very explicit, did you discuss it already?